plugins/calendar-resources/src/components/DayCalendar.svelte

```diff
@@ -501,6 +506,23 @@
       ? rem((heightAD + 0.125) * (adMaxRow <= minAD ? adMaxRow : minAD) + 0.25)
       : rem((heightAD + 0.125) * (adMaxRow <= maxAD ? adMaxRow : maxAD) + 0.25)
   $: showArrowAD = (!minimizedAD && adMaxRow > maxAD) || (minimizedAD && adMaxRow > minAD)
+
+  let dragOnOld: CalendarCell | null = null
+
+  const getMinutes = (e: MouseEvent): number => (e.offsetY >= cellHeight / 2 ? 30 : 0)
+  const dragOver = (e: MouseEvent, day: Date, hourOfDay: number) => {
+    const dragOn: CalendarCell = {
+      day,
+      hourOfDay,
+      minutes: getMinutes(e)
+    }
+    if (dragOn === dragOnOld) return
```

Comment:
This won't work because dragOn is always a new instance and will be equal to itself only. You need to compare individual fields or use other technique.